obsproject/obs-studio
# UI: Add setting for Mixer add-on choice (MixrElixr)

### Description

MixrElixr Emotes is the emote-specific portion of MixrElixr, a popular extension for Mixer chat. Disabled by default. Built in a way to support more extensions in future if needed.

https://crowbartools.com/tools/mixrelixr/

![image](https://user-images.githubusercontent.com/941350/77055768-9850fc80-6a25-11ea-96c4-52cf5fc82a93.png)

### Motivation and Context

This is Mixer's equivalent of BTTV.

https://ideas.obsproject.com/posts/575/

### How Has This Been Tested?
* Connect a Mixer Account
* Choose MixrElixr Emotes in the dropdown
* In the Chat dock, open the Emotes menu
* There should be two sections, one for native emotes, one for MixrElixr emotes

### Types of changes
 - New feature (non-breaking change which adds functionality)

### Checklist:
